templates/16-04/azure.html

```diff
@@ -134,7 +135,7 @@ <h3>
         Microsoft and Canonical joint webinar on 16.04 migration strategies and options
       </h3>
       <p>
-        Microsoft and Canonical partnered on a joint webinar to help you through this evaluation process. The webinar covers the following topics:
+        Microsoft and Canonical partnered on a <a href="https://info.microsoft.com/ww-ondemand-expert-tips-to-easily-migrate-your-ubuntu-stack-to-azure.html">joint webinar</a> to help you through this evaluation process. The webinar covers the following topics:
```

Comment:
It seems we normally add `class="p-link--external"` when it goes to an external website
```suggestion
        Microsoft and Canonical partnered on a <a class="p-link--external" href="https://info.microsoft.com/ww-ondemand-expert-tips-to-easily-migrate-your-ubuntu-stack-to-azure.html">joint webinar</a> to help you through this evaluation process. The webinar covers the following topics:
```